[MWAN] Setup manually or through Connector (#10579)
cloudflare/cloudflare-docs: content/magic-wan/get-started/_index.md

# Get started

> There are two ways of setting up Magic WAN: 

Reviewer: ```suggestion
There are two ways to set up Magic WAN: 
```

> - Through Magic WAN Connector (preferred)

Reviewer: ```suggestion
- Through Magic WAN Connector (preferred).
```
